.env.example

```diff
@@ -6,7 +6,6 @@ NUXT_PUBLIC_NODE_URL=https://mainnet.aeternity.io
 NUXT_PUBLIC_WEBSOCKET_URL=wss://mainnet.aeternity.io/mdw/v2/websocket
 NUXT_PUBLIC_DEX_BACKEND_URL=https://dex-backend-mainnet.prd.service.aepps.com
 NUXT_PUBLIC_NETWORK_NAME=MAINNET
-NUXT_PUBLIC_NETWORK_ID=ae_mainnet
```

Comment:
I didn't find any usages, anyway better to ask node for it

Reply:
Good catch, It is not used now, however will be used for the contract read interaction feature.
I exposed the env.example earlier than it should be.